### PR TITLE
Syntax

### PR DESCRIPTION
**Pull request by jeremybarbet**

### CODE REVIEW

**Comment by osk** [comment]
Nice, especially like keeping the decorators in a new line.

Can we enable some linting rules to enforce this in general?

**Comment by jeremybarbet** [comment]
I will take a look at it! 👊🏻

File: package.json [hunk 1]
```diff
@@ -109,6 +109,7 @@
     "helmet": "3.8.1",
     "hpp": "0.2.2",
     "html-webpack-plugin": "2.30.1",
+    "http-proxy-middleware": "0.17.4",
```
Comment (jeremybarbet) [inline package.json:116]: @osk I removed this rule `'import/no-extraneous-dependencies': 0`, from our default config, not quite sure if it was really useful, does it sound good to you?

```'http-proxy-middleware' should be listed in the project's dependencies, not devDependencies```

Comment (osk) [reply package.json:116]: Yes, sounds good.

**Review by osk — APPROVED** [review]
Failed tests are becaue of some problem with Jest on Heroku CI, unrelated to PR

**Pull request merged**